langium/er-dsl/src/language/MetaModel/Entity.ts

```diff
@@ -62,6 +72,32 @@ export class Entity {
         // return this.simpleString();
     }
 
+    public toPlantUML(): string {
+        let result: string = "";
+
+        result = this.children.map(child => 
+            this.getAggregatedInheritanceType() 
+                ? `${child.name} ->- ${this.getAggregatedInheritanceType()} {${this.name}}`
+                : `${child.name} ->- ${this.name}`
+        ).join('\n\n');
+
+        return result;
+    }
+
+    public toPlantWithAttribute(
+        getDataTypeString: (attribute: Attribute) => string,
+        generateKeyword: (attribute: Attribute) => string
```

Comment:
I wonder why this function takes two functions as arguments, although it likely makes sense later. Even if it makes sense. It is a bit of a code smell.